docs/docs/use-cases/day-2-operations.md

```diff
@@ -62,7 +63,7 @@ and the image must be updated as quickly as possible.
 To do that, change `podtato-head-frontend` as follows:
 
 ```yaml
-{% include "./assets/deployment-new-image.yaml" %}
+{% include "./assets/day-2-operations/deployment-new-image.yaml" %}
```

Comment:
in order to have the code block indented as well in the rendered page, you need to indent the whole code block here. please try out how many spaces are needed, it's not clear to me right now 😅

Reply:
I think it's 3 spaces